telegram_export/utils.py

```diff
@@ -259,3 +260,20 @@ def parse_proxy_str(proxy_str):
     else:
         proxy = (proxy_type, host, port)
     return proxy
+
+
+def format_filename(s):
+    """Take a string and return a valid filename constructed from the string.
+Uses a whitelist approach: any characters not present in valid_chars are
+removed. Also spaces are replaced with underscores.
+
+Note: this method may produce invalid filenames such as ``, `.` or `..`
+When I use this method I prepend a date string like '2009_01_15_19_46_32_'
+and append a file extension like '.txt', so I avoid the potential of using
+an invalid filename.
+
+"""
+    valid_chars = "-_.() %s%s" % (string.ascii_letters, string.digits)
+    filename = ''.join(c for c in s if c in valid_chars)
+    filename = filename.replace(' ', '_')  # I don't like spaces in filenames.
```

Comment:
It is not a personal opinion, actually, I got the exception for the mentioned problem, as you said I can change the code to just remove restricted characters, so it will be all good, right?

Reply:
Yeah, I just used the code I refered, So I will write my own code to fix it again, Thank you for your comments :)